samples/batch/js/node/main.js

```diff
@@ -0,0 +1,333 @@
+            }
+         //   console.log("***************************************************************  response "+JSON.stringify(body));
+            console.log("***************************************************************************");
+
+            console.log(body);
+            //content.decode('utf-8')
+        });
+        
+     }
+   }
+  }
+};
+        
+//_transcribe();
+//_deleteAllTranscriptions();
+
+function _transcribe()
+{
+    
+    // Configure API key authorization: apiKeyHeader
+    var apiKeyHeader = defaultClient.authentications['apiKeyHeader'];
+    apiKeyHeader.apiKey = '';
+    // Configure API key authorization: apiKeyQuery
+    var apiKeyQuery = defaultClient.authentications['apiKeyQuery'];
+    apiKeyQuery.apiKey = '';
+
+    //Create API Instance
+    //var apiInstance = new SpeechToTextApiV30.DefaultApi();
+
+    const subscriptionKey = "{your-subscription-key-here}";
+    const serviceRegion = "centralindia"; // e.g., "westus"
+    defaultClient.basePath = 'https://centralindia.api.cognitive.microsoft.com/speechtotext/v3.0'.replace(/\/+$/, '');
+
+    // Uncomment this block to transcribe one file.
+    //var opts = _transcribeFromSingleBlob(RECORDINGS_BLOB_URI);
+
+    // Uncomment this block to use custom models for transcription.
+    // transcription_definition = transcribe_with_custom_model(api, RECORDINGS_BLOB_URI, properties)
+
+    // Uncomment this block to transcribe all files from a container.
+    var opts = _transcribeFromContainer(RECORDINGS_CONTAINER_URI);
+    
+    apiInstance.createTranscription(opts, function(error, data, response) {
+        if (error) {
+          console.error(error);
+        } 
+        else {
+          var completed = false;
+          var tid = data.self.substring(data.self.lastIndexOf("/")+1);
+          console.log("Transaction ID is :"+tid);
+          apiInstance.getTranscription(tid,callbackGetTrans);
+          
+        }
+      });
+    
+}//end of _transcribe()
+
+
+function _deleteAllTranscriptions(){
+    
+    //Delete all transcriptions associated with your speech resource.
+    console.log("Deleting all existing completed transcriptions.")
+    // Configure API key authorization: apiKeyHeader
+    var apiKeyHeader = defaultClient.authentications['apiKeyHeader'];
+    apiKeyHeader.apiKey = '';
+    // Configure API key authorization: apiKeyQuery
+    var apiKeyQuery = defaultClient.authentications['apiKeyQuery'];
+    apiKeyQuery.apiKey = '';
+
+    //Create API Instance
+    //var apiInstance = new SpeechToTextApiV30.DefaultApi();
+
+    const subscriptionKey = "{your-subscription-key-here}";
+    const serviceRegion = "centralindia"; // e.g., "westus"
+    defaultClient.basePath = 'https://centralindia.api.cognitive.microsoft.com/speechtotext/v3.0'.replace(/\/+$/, '');
+
+    // get all transcriptions for the subscription
+     var opts = { 
+        'skip': 56, // Number | Format - int32. Number of transcriptions that will be skipped.
```

Comment:
why these numbers?

Reply:
@chlandsi - Have incorporated the changes in the latest code for main.js.